### PR TITLE
Autoproxy improvements

### LINKED ISSUE #190: Make timeout configurable for `pk;ap latch`
A friend of mine spends most of the time fronting in their system, so they make heavy use of latch mode. Usually, when the latch timeout passes (e.g., the next morning), the same system member is still fronting. They've been getting frustrated by having to remember to re-initiate the latch every day. Would it be possible to make the timeout configurable, possibly with an option to never time out?

### PR DESCRIPTION
**Pull request by dev-kittens**
* Disable front/latch autoproxy per-member
* configurable latch timeout duration (closes #190)
* Disable autoproxy per Discord account
* Some documentation fixes

### CODE REVIEW

**Comment by dev-kittens** [comment]
that's not a conflict, github
image

File: PluralKit.Core/Database/Migrations/12.sql [hunk 1]
```diff
@@ -0,0 +1,10 @@
+-- SCHEMA VERSION 12: <insert date> --
+-- Changelog:
+-- * Add disabling front/latch autoproxy per-member
+-- * Add configurable latch timeout duration
+-- * Add disabling autoproxy per-account
+
+alter table members add column disable_autoproxy bool not null default false;
+alter table systems add column latch_timeout int not null default 6;
```
Comment (xSke) [inline PluralKit.Core/Database/Migrations/12.sql:8]: Perhaps leave this null by default and have a fallback defined in code? That way that can be changed separately

**Comment by dev-kittens** [comment]
welp, merge conflicts destroyed this branch. going to close this PR and recreate it

File: PluralKit.Core/Database/Migrations/12.sql (hunk 1 above)
Comment (dev-kittens) [reply PluralKit.Core/Database/Migrations/12.sql:8]: I updated this in the new PR.

**Pull request closed without merging**